docs/howto_config.md

```diff
@@ -10,19 +10,21 @@ The MicroShift configuration file must be located at `~/.microshift/config.yaml`
 The format of the `config.yaml` configuration file is as follows.
 
 ```yaml
-subjectAltNames:
-  - ""
-nodeName: ""
-nodeIP: ""
-url: ""
+cluster:
+  name: ""
+  baseDomain: ""
 network:
   clusterNetwork:
     - cidr: ""
   serviceNetwork:
     - ""
   serviceNodePortRange: ""
-dns:
-  baseDomain: ""
+node:
+  hostnameOverride: ""
```

Comment:
I believe @fzdarsky picked this one to align with the kubelet parameter of the same meaning. It's long and not nice, but descriptive of the purpose.

I am ok with both options though.

Reply:
Was thinking about just "name", but "hostnameOverride" would have two advantages:
1. It makes explicit what the parameter is eventually used for, namely being passed to the kubelet via the "hostnameOverride" parameter. The kubelet otherwise defaults the node name to host name (like we do).
2. It makes it more intuitive that this param should be left empty unless there's a reason to override the default behavior.

Reply:
OK, I think that makes sense.

Reply:
I'm up for `hostnameOverride` too. It's a well-known param for kubelet.